src/main/java/teammates/logic/core/FeedbackResponseCommentsLogic.java

```diff
@@ -237,99 +234,148 @@ public boolean isNameVisibleTo(FeedbackResponseCommentAttributes comment, Feedba
             currentUserTeam = roster.getStudentForEmail(userEmail).team;
         }
         
-        for (FeedbackParticipantType type : showNameTo) {
-            if (type == FeedbackParticipantType.GIVER && userEmail.equals(response.giver)) {
-                return true;
-            } else if (type == FeedbackParticipantType.INSTRUCTORS && roster.getInstructorForEmail(userEmail) != null) {
-                return true;
-            } else if (type == FeedbackParticipantType.RECEIVER && userEmail.equals(response.recipient)) {
-                return true;
-            } else if (type == FeedbackParticipantType.OWN_TEAM_MEMBERS && responseGiverTeam.equals(currentUserTeam)) {
-                return true;
-            } else if (type == FeedbackParticipantType.RECEIVER_TEAM_MEMBERS
-                    && responseRecipientTeam.equals(currentUserTeam)) {
-                return true;
-            } else if (type == FeedbackParticipantType.STUDENTS && roster.getStudentForEmail(userEmail) != null) {
-                return true;
-            }
-        }
-        return false;
+        return doesVisibilityRecipientTypeMatchUserDetails(response, userEmail, roster, showNameTo, responseGiverTeam,
+                responseRecipientTeam, currentUserTeam);
     }
     
     /**
      * Verify whether the comment is visible to certain user
+     * Does not take individual instructor privileges into account
      * @return true/false
      */
     public boolean isResponseCommentVisibleForUser(String userEmail, String courseId, UserType.Role role,
-            String section, StudentAttributes student, Set<String> studentsEmailInTeam,
-            FeedbackResponseAttributes response, FeedbackQuestionAttributes relatedQuestion,
-            FeedbackResponseCommentAttributes relatedComment, InstructorAttributes instructor) {
-        if (response == null || relatedQuestion == null) {
+            StudentAttributes student, Set<String> studentsEmailInTeam, FeedbackResponseAttributes relatedResponse,
+            FeedbackQuestionAttributes relatedQuestion, FeedbackResponseCommentAttributes comment) {
```

Comment:
This method never used `section` or `instructor` in the first place; this might indicate it was supposed to have taken instructor privileges into account at one point, but was never done/removed.

All instructor entry points for feedback response comments (InstructorCommentsPage, InstructorStudentRecordsPage, InstructorFeedbackResultsPage) are covered by checks to visibility of responses however, and comments are not displayed if their corresponding responses are not visible.